documentation/examples/20-7guis/06-7guis-circles/App.svelte

```diff
@@ -69,9 +69,10 @@ radius of the selected circle.
 	<button on:click={() => travel(-1)} disabled={i === 0}>undo</button>
 	<button on:click={() => travel(+1)} disabled={i === undoStack.length - 1}>redo</button>
 </div>
-
-<svg on:click={handleClick}>
+<!-- svelte-ignore a11y-click-events-have-key-events -->
+<svg on:click={handleClick} role='button' tabindex='0'>
```

Comment:
I think this is another warning that would be better to ignore for now -- adding `role` and `tabindex` doesn't actually make this more accessible.